packages/network/src/api/image/requests.ts

```diff
@@ -169,12 +205,24 @@ function getImageLabel(options: AddImageOptions): TranslationObject | undefined
       nl: `Videoframe ${options.frameIndex}`,
     };
   }
-  return {
-    en: options.firstShot ? 'Close Up (part)' : 'Close Up (damage)',
-    fr: options.firstShot ? 'Photo Zoomée (partie)' : 'Photo Zoomée (dégât)',
-    de: options.firstShot ? 'Gezoomtes Foto (Teil)' : 'Close Up (Schaden)',
-    nl: options.firstShot ? 'Nabij (onderdeel)' : 'Nabij (schade)',
-  };
+  if (options.uploadType === ImageUploadType.PART_SELECT_SHOT) {
+    const partsTranslation = options.vehicleParts.map((part) => vehiclePartLabels[part]);
+    return {
+      en: `Damage on ${partsTranslation.map((part) => part.en).join(', ')}`,
+      fr: `Dommages sur ${partsTranslation.map((part) => part.en).join(', ')}`,
+      de: `Schaden an ${partsTranslation.map((part) => part.en).join(', ')}`,
+      nl: `Schade aan ${partsTranslation.map((part) => part.en).join(', ')}`,
```

Comment:
Avoid spaces in file name : `close_up_${partsTranslation.map((part) => part.en).join('_')}`